client/reader/following-edit/index.jsx

```diff
@@ -348,7 +385,8 @@ var FollowingEdit = React.createClass( {
 				{ this.state.isAttemptingFollow && ! this.state.lastError ? <SubscriptionPlaceholder key={ 'placeholder-add-feed' } /> : null }
 				{ subscriptionsToDisplay.length === 0 && this.props.search ?
 					<NoResults text={ this.translate( 'No subscriptions match that search.' ) } /> :
-				<InfiniteList role="main"
+
+				<InfiniteList role="main" className="followed-sites-list"
```

Comment:
the `followed-sites-list` also doesn't match the component. Should be `following-edit__sites`